app/views/partials/alerts/form_field_string_value.scala.html

```diff
@@ -0,0 +1,19 @@
+@(stream: org.graylog2.restclient.models.Stream, conditionDecorator: views.helpers.FieldStringValueAlertConditionDecorator)
+
+<form action="@conditionDecorator.getFormAction(stream.getId)" method="POST" class="alert-type-form alert-type-form-field-string-value" id="@conditionDecorator.getFormId">
+    <div class="well well-small">
+        <h2 style="margin-bottom: 10px;">
+            <i class="fa fa-bell-alt"></i>
+            @conditionDecorator.getFormTitle
```

Comment:
Adding the title to the inline form is inconsistent with the other alert condition types.